tests/rptest/redpanda_cloud_tests/cloud_self_test.py

```diff
@@ -7,6 +7,7 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0
 
+import time
```

Comment:
redundant?

Reply:
@savex - yes good call, I had used `time` for debugging. Fixed in push https://github.com/redpanda-data/redpanda/commit/ac07e97aa77d72289740664cce3f9852e59894f8